complex objects/cargo/list/items.h

```diff
@@ -15,6 +14,8 @@ int64_t* _Nullable item_get_due_date(const struct item* _Nonnull item);
 const void item_set_due_date(struct item* _Nonnull item, int64_t due_date);
 const size_t item_get_is_complete(const struct item* _Nonnull item);
 const void item_set_is_complete(struct item* _Nonnull item, size_t is_complete);
```

Comment:
Worth thinking about whether you want to reflect a completion date here, or just in the Mentat version. This API can be lossy if you like.

Reply:
Oops, forgot to update these functions in the header. Thanks for spotting.